Nodejs/Product/Nodejs/LanguagePreferences.cs

```diff
@@ -47,12 +48,18 @@ class LanguagePreferences : IVsTextManagerEvents2 {
             return VSConstants.S_OK;
         }
 
-        public int OnUserPreferencesChanged2(VIEWPREFERENCES2[] viewPrefs, FRAMEPREFERENCES2[] framePrefs, LANGPREFERENCES2[] langPrefs, FONTCOLORPREFERENCES2[] colorPrefs) {
-            if (langPrefs != null && langPrefs.Length > 0 && langPrefs[0].guidLang == this._preferences.guidLang) {
-                _preferences.IndentStyle = langPrefs[0].IndentStyle;
-                _preferences.fAutoListMembers = langPrefs[0].fAutoListMembers;
-                _preferences.fAutoListParams = langPrefs[0].fAutoListParams;
-                _preferences.fHideAdvancedAutoListMembers = langPrefs[0].fHideAdvancedAutoListMembers;
+        public int OnUserPreferencesChanged4(VIEWPREFERENCES3[] pViewPrefs, LANGPREFERENCES3[] pLangPrefs, FONTCOLORPREFERENCES2[] pColorPrefs) {
+            IVsTextManager4 textMgr = (IVsTextManager4)NodejsPackage.Instance.GetService(typeof(SVsTextManager));
```

Comment:
We don't null check anywhere, and I wouldn't expect it to ever be null (except maybe inside of some of the mock test cases?)